include/wlr/types/wlr_screencopy_v1.h

```diff
@@ -27,9 +26,14 @@ struct wlr_screencopy_manager_v1 {
 	void *data;
 };
 
+struct wlr_screencopy_v1_client {
+	struct wl_list frames; // wlr_screencopy_frame_v1::link
+	struct wl_list damages;
+};
+
 struct wlr_screencopy_frame_v1 {
 	struct wl_resource *resource;
-	struct wlr_screencopy_manager_v1 *manager;
+	struct wlr_screencopy_v1_client *connection;
```

Comment:
All of these should be renamed to `client` as well.

Reply:
This member can be changed to "client", but arguments and/or variable clash with wl_client parameters in some places. I could rename those wl_client.